src/providers/storage/bingImageSearch.json

```diff
@@ -3,13 +3,22 @@
     "title": "${strings.connections.providers.bing.options}",
     "required": ["apiKey","query"],
     "properties": {
+        "endpoint": {
+            "type": "string",
+            "title": "Endpoint",
+            "description": "The endpoint from your Bing Search Azure resource",
+            "default": "https://api.bing.microsoft.com/",
+            "pattern": "^https?\\\\://[a-zA-Z0-9\\\\-\\\\.]+\\\\.[a-zA-Z]{2,3}(/\\\\S*)?$"
+        },
         "apiKey": {
             "type": "string",
-            "title": "${strings.connections.providers.bing.apiKey}"
+            "title": "${strings.connections.providers.bing.apiKey}",
+            "description": "The key from your Bing Search Azure resource"
         },
         "query": {
             "type": "string",
-            "title": "${strings.connections.providers.bing.query}"
+            "title": "${strings.connections.providers.bing.query}",
+            "description": "The "
```

Comment:
Is this supposed to more that just "The"?

Reply:
Yes, fixed.